server/utils/referrals/showReferralUtils.ts

```diff
@@ -49,6 +49,27 @@ export default class ShowReferralUtils {
     ]
   }
 
+  static subNavigationItems(
+    currentPath: Request['path'],
+    currentSection: 'referral' | 'risksAndNeeds',
+    referralId: Referral['id'],
+  ): Array<MojFrontendNavigationItem> {
+    const paths = currentPath.startsWith(assessPathBase.pattern) ? assessPaths : referPaths
+
+    return [
+      {
+        active: currentSection === 'referral',
+        href: paths.show.personalDetails({ referralId }),
+        text: 'Referral details',
+      },
+      {
+        active: currentSection === 'risksAndNeeds',
```

Comment:
Maybe less robust (in case the path has some unexpected characters in it), but could we determine this based on the `currentPath` matching the `href` within the same item, removing the need for the `currentSection` parameter?

Reply:
Happy to try an alternative, can catch up and give it a try tomorrow.